boards/arm/bl5340_dvk/doc/index_bl5340pa.rst

```diff
@@ -0,0 +1,454 @@
+.. _bl5340pa_dvk:
+
+Laird Connectivity BL5340PA DVK
+###############################
+
+Overview
+********
+The BL5340PA Development Kit provides support for the Laird
+Connectivity BL5340PA module which is powered by a dual-core Nordic
+Semiconductor nRF5340 ARM Cortex-M33F CPU and a Nordic Semiconductor

```

Comment:
Isn't just `M33` not `M33F`

Reply:
I believe this was copied from Nordic thingy53.  Do you think source is incorrect?

Reply:
Datasheet reads as follows.

nRF5340 SoC Product Brief Version 2.2
For more information please visit: nordicsemi.com/nRF5340
Key features
 High-performance application processor
 128/64 MHz Arm Cortex-M33 with
FPU and DSP instructions
 1 MB Flash + 512 KB RAM
 8 KB 2-way set associative cache

 Fully-programmable network processor
 64 MHz Arm Cortex-M33 with
2 KB instruction cache
 256 KB Flash + 64 KB RAM

Reading this as the application core being 33F and the network core plain 33.

Reply:
Hmm yes, seems ok.